clang/test/SemaHIP/builtins-amdgcn-raw-buffer-atomic-add.hip

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -fsyntax-only -triple amdgcn -target-cpu gfx90a -verify %s -fcuda-is-device
+// RUN: %clang_cc1 -fsyntax-only -triple x86_64 -aux-triple amdgcn -verify %s
+
+typedef _Float16 __attribute__((ext_vector_type(2))) float16x2_t;
```

Comment:
Did you add this test for later PR?

Reply:
If I understand https://clang.llvm.org/docs/LanguageExtensions.html#half-precision-floating-point , the difference between `fp16` and `_Float16` is not in the representation, but in the promotion rules. Am I right ?